### DISCUSSION TITLE
Use local workflow with relative path

### DISCUSSION THREAD

**Discussion by alvaromartmart** [community/community#25799] [Actions]
<p>Hi,</p>
<p>Is it possible to re-use a local workflow in the same repository using a relative path?<br>
I tried</p>
<pre><code class="lang-auto">uses: ./.github/workflows/my-local-workflow.yml
</code></pre>
<p>But it complains about the <code>@tag</code> missing, and I’m not sure this is supposed to work anyway. Any tips?</p>

**Comment by Simran-B** [comment]
<p>What happens if you add the branch name the main workflow is on as tag, like <code>@main</code>?</p>

**Comment by alvaromartmart** [comment]
<p>That works, but then I’m forced to use the specific branch where I’m developing the workflow, and remember to change it to <code>main</code> before merging, which is a bit inconvenient. I was wondering if there was an alternative</p>

**Comment by Simran-B** [comment]
<p>Not sure. It cannot be dynamic because of security reasons but maybe <code>HEAD</code> works as tag? I have doubts that such Git aliases are exposed, however (again because of security concerns).</p>

**Comment by alvaromartmart** [comment, marked as answer]
<p>In case someone’s interested, I ended up using a local action instead of local workflow. The only disadvantage is that you need a previous <code>checkout</code> step before you can use the local action.</p>
